lib/uiauto-client.js

```diff
@@ -0,0 +1,237 @@
+// The Command Proxy relays UIAuto message to and from Appium. It is also the
+// UIAuto facade for Appium.
+//
+// The message route is the following:
+// Appium <--> Command Proxy <--> Instruments
+// The medium between Instruments and Command Proxy is the command-proxy-client
+// script.
+//
+// Command Proxy --> Instruments message format: {cmd:"<CMD>"}
+//
+// Instruments --> Command Proxy message format:
+// <one char message type>,<stringified json data>
+// <stringified json data> format:
+// {status:<status>, value:<result>}
+
+import log from './logger';
+import _fs from 'fs';
+import through from 'through';
+import net from 'net';
+import _mkdirp from 'mkdirp';
+import path from 'path';
+import Promise from 'bluebird';
+import _ from 'lodash';
+
+let fs = {
+  unlink: Promise.promisify(_fs.unlink)
+};
+let mkdirp = Promise.promisify(_mkdirp);
+
+
+var MORE_COMMAND = '#more';
+var MESSAGE_TYPES = ['error', 'no data', 'regular', 'chunk', 'last chunk'];
```

Comment:
`var`s here. Also, this is the same array as in bin/command-proxy-client, we might want to have it be in one place